docs/specification/draft/client/elicitation.mdx

```diff
@@ -321,12 +526,212 @@ Servers should handle each state appropriately:
+
+In this pattern:
+
+1. The server directs users to a secure web page (served over HTTPS)
+2. The page presents a branded form UI on a domain the user trusts
+3. Users enter sensitive credentials directly into the secure form
+4. The server stores credentials securely, bound to the user's identity
+5. Subsequent MCP requests use these stored credentials for API access
+
+This approach ensures that sensitive credentials never pass through the MCP client or any intermediate MCP servers, reducing the risk of exposure through client-side logging or other attack vectors.
+
+### URL Elicitation for OAuth Flows
+
+URL elicitation enables a pattern where MCP servers act as OAuth clients to third-party resource servers.
+The "downstream authorization" enabled by url elicitation is fundamentally separate from [MCP authorization](../basic/authorization).
+
+<Note>
+  This distinction is critical to understand. URL elicitation is *not* about
+  authorizing (or re-authorizing) the MCP client to access the MCP server. It
+  *can* be used to enable the MCP server to access third-party resources or APIs
+  on behalf of the user, without exposing sensitive credentials or tokens to the
+  MCP client.
+</Note>
+
+#### Understanding the Distinction
+
+- **MCP Authorization**: Required OAuth flow between the MCP client and MCP server (covered in the [authorization specification](../basic/authorization))
+- **Downstream Authorization**: Optional authorization between the MCP server and a third-party resource server, initiated via url elicitation
+
+In downstream authorization, the server acts as both:
+
+- An OAuth resource server (to the MCP client)
+- An OAuth client (to the third-party resource server)
+
+Consider this hypothetical scenario:
+
+- Claude Desktop (the MCP client) connects to Bob's Productivity Server (an HTTPS MCP server)
+- Bob's Productivity Server integrates with Dropbox and Google APIs
+- When Claude calls a tool that requires Dropbox access, the MCP server needs Dropbox credentials
+
+The critical security requirements are:
+
+1. **The Dropbox token MUST NOT transit through Claude Desktop**: The client must never see third-party credentials
+2. **The MCP server MUST NOT use its Claude access token for Dropbox**: That would be [token passthrough](../basic/security_best_practices#token-passthrough), which is forbidden
+3. **The user must authorize the MCP server directly**: The interaction happens outside the MCP protocol
+
+The tokens from these flows are never mixed or passed through. The MCP client's token is for the MCP server only, and the third-party tokens never leave the MCP server.
+
+<Note>
+  For more background, read the [token passthrough
+  section](../basic/security_best_practices#token-passthrough) of the Security
+  Best Practices document to understand why MCP servers cannot act as
+  pass-through proxies.
+</Note>
+
+#### Implementation Pattern
+
+When implementing downstream authorization via url elicitation:
```

Comment:
It's worth calling out that this is _one of_ the possible patterns. The server can also ask for API keys or other user details.